### DISCUSSION TITLE
Error on login

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#847] [Q&A (Community Support)] [answered]
Hello, I tried to log in to the page today and got the problem shown in the screenshot. 
The only setup I did was update to the most recent version of Panopticon yesterday, and everything functioned as it should have.

**Comment by nikosdion** [comment]
The actual error message is missing from your post.

**Comment by komir2004** [comment]
[Error] https://panopticon.*******.com/index.php?view=login&task=login#trace-box-1
HTTP 500
Call to a member function getId() on null

**Reply by nikosdion** [reply, marked as answer]
This is a bug which only occurs when you use the wrong username.

**Reply by komir2004** [reply]
The actual "bug" is the cause of the problem; thank you for helping.
